app/controllers/concerns/foreman/controller/registration.rb

```diff
@@ -19,18 +19,20 @@ def global_registration_vars
     host_group = Hostgroup.authorized(:view_hostgroups).find(params['hostgroup_id']) if params["hostgroup_id"].present?
     operatingsystem = Operatingsystem.authorized(:view_operatingsystems).find(params['operatingsystem_id']) if params["operatingsystem_id"].present?
 
+    context = {
+      user: User.current,
+      auth_token: User.current.jwt_token!(expiration: 4.hours.to_i),
```

Comment:
Any idea why we are using 4.hours.to_i, does 4 hours has a significance or it is a random time frame?

Reply:
It was there before, just moving things around. However, I would agree that it's ugly that it's hardcoded and not even a constant.

Reply:
makes sense :+1:

Reply:
a bit of offtopic to this PR: We could extract that to be configurable, however is that necessary? Personally I'd wait until someone asks for it, the first token is configurable already. We have enough settings already. The other option would be to base on the expiration set for the first request. However do not that these two expirations are not the same thing. If you think it's valuable, please describe how we should set this and link the issue to https://projects.theforeman.org/issues/30440.